### PR TITLE
Shows `ERROR` in red when a command fails

### PR DESCRIPTION
**Pull request by AlianBenabdallah**
Closes: [#2431](https://github.com/informalsystems/ibc-rs/issues/2431)

## Description

The output of CLI can be confusing (see linked issue). This PR shows the status of the output in green if the command succeeds or in red if it fails.

______

### PR author checklist:
- [ ] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests: integration (for Hermes) or unit/mock tests (for modules). 
- [ ] Linked to GitHub issue.
- [ ] Updated code comments and documentation (e.g., `docs/`).

### Reviewer checklist:

- [ ] Reviewed `Files changed` in the GitHub PR explorer.
- [ ] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

**Comment by romac** [comment]
Nice! Looks like there are comments from the `ali/fix_oldest_sequence` branch, can you please remove those and only keep the last one?

File: guide/src/telemetry.md [hunk 1]
```diff
@@ -51,8 +51,8 @@ The following table describes the metrics currently tracked by the telemetry ser
 | `acknowledgement_count`      | Number of WriteAcknowledgement events processed      | `u64` Counter       |
 | `cleared_send_packet_count`    | Number of SendPacket events processed during the initial and periodic clearing | `u64` Counter   |
 | `cleared_acknowledgment_count` | Number of WriteAcknowledgement events processed during the initial and periodic clearing | `u64` Counter   |
-| `oldest_sequence`            | The sequence number of the oldest SendPacket event observed without its corresponding WriteAcknowledgement event. If this value is 0, it means Hermes observed a WriteAcknowledgment event for all the SendPacket events | `u64` ValueRecorder |
```
Comment (adizere) [inline guide/src/telemetry.md:54]: Do these changes belong in this PR?

File: relayer/src/worker/packet.rs [hunk 2]
```diff
@@ -267,17 +267,35 @@ fn acknowledgment_metrics(path: &Packet, summary: &RelaySummary) {
 #[cfg(feature = "telemetry")]
 fn timeout_metrics(path: &Packet, summary: &RelaySummary) {
     use ibc::events::IbcEvent::TimeoutPacket;
-    let count = summary
+    let mut count = 0;
```
Comment (adizere) [inline relayer/src/worker/packet.rs:270]: This likely belongs in a separate PR.

File: relayer-cli/src/conclude.rs [hunk 3]
```diff
@@ -72,7 +73,11 @@ pub fn exit_with(out: Output) -> ! {
     if json() {
         println!("{}", serde_json::to_string(&out.into_json()).unwrap());
     } else {
-        println!("{}: {}", out.status, out.result);
+        let status = match out.status {
+            Status::Success => style("SUCCESS").green(),
+            Status::Error => style("ERROR").red(),
+        };
+        println!("{} {}", status, out.result);
```
Comment (adizere) [inline relayer-cli/src/conclude.rs:80]: Neat!

**Comment by AlianBenabdallah** [comment]
It looks like I mixed 2 PR. I'll fix this in a moment

File: relayer/src/worker/packet.rs [hunk 4]
```diff
@@ -267,17 +267,35 @@ fn acknowledgment_metrics(path: &Packet, summary: &RelaySummary) {
...
+    summary
         .events
         .iter()
         .filter(|e| matches!(e, TimeoutPacket(_)))
```
Comment (seanchen1991) [inline relayer/src/worker/packet.rs:274]: To make this slightly more readable.
```suggestion
        .filter(|event| matches!(event, TimeoutPacket(_)))
```

File: relayer/src/worker/packet.rs [hunk 5]
```diff
@@ -267,17 +267,35 @@ fn acknowledgment_metrics(path: &Packet, summary: &RelaySummary) {
...
+                        u64::from(packet.sequence),
+                        height,
+                        &path.src_chain_id,
+                        &path.src_channel_id,
+                        &path.src_port_id,
+                        &path.dst_chain_id,
+                    );
+                }
+                None => {}
+            }
```
Comment (seanchen1991) [inline relayer/src/worker/packet.rs:290]: I think this would be slightly cleaner using `if let` syntax so that the `None` case doesn't need to be explicitly accounted for. 
```suggestion
            if let Some(packet) = e.packet() {
                telemetry!(
                    record_ack_history,
                    u64::from(packet.sequence),
                    height,
                    &path.src_chain_id,
                    &path.src_channel_id,
                    &path.src_port_id,
                    &path.dst_chain_id,
                );
            }
```

**Review by seanchen1991 — COMMENTED** [review]
I saw you mention that you mixed two PRs together here, but I figured I'd leave some nits that you can apply after you've split this work into two PRs.
